loopchain/peer/peer_service.py

```diff
@@ -169,7 +156,7 @@ def __get_channel_infos(self):
                     message=loopchain_pb2.GetChannelInfosRequest(
                         peer_id=self.__peer_id,
                         peer_target=self.__peer_target,
-                        group_id=self.group_id),
+                        group_id=self.__peer_id),
```

Comment:
This group_id should be deleted.

Reply:
`group_id` is partially removed here. Group ids related to grpc proto will be updated at the next pull request.